WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -110,8 +110,12 @@
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons is the recommended icon font." />
+
+        <!-- https://github.com/CommunityToolkit/Windows/issues/240 -->
+        <!-- External toolkit SettingsExpander does not pass Axe testing-->
         <toolkit:SettingsExpander
             Grid.Row="1"
+            AutomationProperties.AccessibilityView="Raw"
```

Comment:
>AutomationProperties.AccessibilityView="Raw" [](http://example.com/codeflow?start=12&length=44)

This seems like the wrong approach. We should update the test to skip this control instead of moving it to the Raw view.

Moving this control to the Raw view will result in the app not working correctly in Narrator and other Assistive Technologies.